tasks/containers/TaskWorker.js

```diff
@@ -79,14 +79,32 @@ export default compose(
 
```

Comment:
oh shit, i might have figured this out.

this line is bad: https://github.com/root-systems/cobuy/blob/feature/task-completion/tasks/containers/TaskWorker.js#L139

it should be `gte(__, 1)`, where `__` is a [special placeholder](http://ramdajs.com/0.21.0/docs/#__) when you need to curry in a particular order. the reason is because [`gte`](http://ramdajs.com/0.21.0/docs/#gte) expects arguments in `x > y` order, which means we are currying `x` as `1`, so it only works when it shouldn't work. (oops!)

another tip that helped me here (i found this while working on #141) was using [`tap`](http://ramdajs.com/0.21.0/docs/#tap) inside a ramda `pipe` like this.